tests/test_progress_tracker.py

```
@@ -0,0 +1,354 @@
+        tracker = ProgressTracker(total_estimated_nodes=1000)
+        tracker.start()
+
+        # 模拟一些进度
+        time.sleep(0.1)  # 等待一点时间
+        tracker.update(5, 100, tokens_used=500)
+
+        # show_stats 不应该抛出异常
+        try:
+            tracker.show_stats()
+            success = True
+        except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)